inc/commondbtm.class.php

```diff
@@ -1198,6 +1198,7 @@ function add(array $input, $options = [], $history = true) {
     * @param boolean $history do history log ? (true by default)
     *
     * @return integer the new ID of the clone (or false if fail)
+    * @deprecated x.x.x Use the {@link \Glpi\Features\Clonable} trait instead
```

Comment:
I wonder if those should not be just removed; they really should not exists on objects that do not use the trait; and have the same names...

Reply:
ping @cedric-anne @orthagh

Reply:
I've removed them, any object using clone features must use the trait already; it should not change anything (pushed here by mistake)

Reply:
We may keep them to handle edge cases (plugins that are clonable objects that are not using the clonable trait).